### DISCUSSION TITLE
Move Diagnostics from Info to Debug or Trace

### DISCUSSION THREAD

**Discussion by belmix** [coinfoundry/miningcore#942] [Ideas]
![image](https://user-images.githubusercontent.com/4902701/133704696-9aadea2f-bfab-479a-acb3-a90021c60599.png)
**### Strongly clogs the log and monitoring of work**

**Comment by oliverw** [comment]
I don't think its possible to alter the level of log messages generated in code not under your control. What's possible is to redirect them, which is already possible by enabling the dedicated api.log.
